src/main/java/com/bullhorn/dataloader/util/TemplateUtil.java

```diff
@@ -54,14 +55,33 @@ private void writeToCsv(String entity, ArrayList<String> headers, ArrayList<Stri
     }
 
     protected void populateDataTypes(String entity, Set<Field> metaFieldSet, ArrayList<String> headers, ArrayList<String> dataTypes) {
```

Comment:
Since we now need entity class information, we should stop passing in an entity string name (which we are getting rid of everywhere eventually).  Pass in the `dataloader/meta/Entity.java` enum instead everywhere that the String name is being used currently.  Go from string name to Enum in the TemplateService using: `Entity.fromString(entityName)`